spec/support/activity_helpers.rb

```diff
@@ -51,4 +51,10 @@ def page_displays_an_activity(activity_presenter:)
     expect(page).to have_content t("summary.label.activity.aid_type")
     expect(page).to have_content activity_presenter.aid_type
   end
+
+  def parent_is_newton?(parent)
```

Comment:
I found this method's name and argument name very confusing. In the context of this method, `parent` is just an activity that is being tested.

Could we use `associated_fund` for the naming instead?

We already use the notion of an associated fund https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/models/activity.rb#L218

 `associated_fund` returns itself if the activity is a fund, or it goes up the lineage until it finds the fund.

What do you think about this version?

`def associated_fund_is_newton?(activity)
  return if activity.nil?

  activity.associated_fund.roda_identifier_fragment == "NF"
end
```